src/mixins/DynamicChildrenMixin.js

```diff
@@ -26,29 +26,45 @@ const DynamicChildrenMixin = types.model({
     };
 
     return {
+      updateWithDynamicChildren(data) {
+        const list = (self.children ?? []).concat(prepareDynamicChildrenData(data));
+
+        self.children = list;

```

Comment:
This setter in the shared store will lock, but I don't see a corresponding unlock in this mixin. What controls the unlock of the shared store here?

Reply:
it's intended. the store is unlocked by default, and it's not supposed to be unlocked after children are set. otherwise, other taxonomies will do the same work over and over again